.github/workflows/test-k3s.yml

```diff
@@ -17,5 +17,7 @@ jobs:
       - name: Make Packages
```

Comment:
IDK if it's necessary to create a convention, unless we are able to automate checks for it.

I added the quotes on a whim, since it felt like the name stood out better in the IDE since they turn green when they are wrapped in quotes. I'm perfectly fine with pulling out the quotes, or adding quotes to the other yaml files that i missed.